design-system/react/src/components/ContentLoader/defs.ts

```diff
@@ -0,0 +1,21 @@
+import type { IContentLoaderProps } from 'react-content-loader';
+
+import type { Components } from '~/defs';
+import type { CreateComponent } from '~/utils';
+
+export type ContentLoaderProps = IContentLoaderProps & {
+  isFullWidth?: boolean;
+};
+
+export type ContentLoaderRectProps = React.SVGProps<SVGRectElement> & {
+  stickX?: 'right' | 'left';
+  stickY?: 'top' | 'bottom';
+};
+
+export type ContentLoaderDef = CreateComponent<{
+  type: 'div';
+  component: Components.ContentLoader;
+  element: HTMLDivElement;
+  props: ContentLoaderProps;
+  styles: 'root';
```

Comment:
you need to omit the `as` prop for this component, since it's can't be polymorphic